client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,44 @@
+.scoreboard {
+  box-sizing: border-box;
+  background: #ffffff;
+  color: #ffffff;
+  border: 2px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.scoreboard table {
+  padding: none;
+  border:  none;
+  border-collapse: collapse;
+  display: block;
+  overflow-x: scroll;
+  overflow-y: scroll;
+}
+
+.scoreboard .firstRow {
+  position: sticky;
```

Comment:
Set z-index: 1, so that as you scroll down the top-left cell shows in front of the other `.firstCol` cells.